### ISSUE TITLE
Problems with libraries in parser version 1.1.2

### ISSUE THREAD

**Issue by sichvoge** [raml-org/raml-js-parser-2#459]
The following example [here](https://github.com/raml-org/raml-examples/tree/master/others/world-music-api) worked perfectly fine with the previous parser version 1.1.1, but not with 1.1.2 anymore. 

The following error is what I get:

```json
{
 "parserErrors": [
   {
     "code": 11,
     "message": "inheriting from unknown type",
     "path": "api.raml",
     "range": {
       "start": {
         "line": 66,
         "column": 12,
         "position": 1495
       },
       "end": {
         "line": 66,
         "column": 28,
         "position": 1511
       }
     },
     "isWarning": false,
     "trace": [
       {
         "code": 11,
         "message": "inheriting from unknown type",
         "path": "api.raml",
         "range": {
           "start": {
             "line": 168,
             "column": 0,
             "position": 4383
           },
           "end": {
             "line": 168,
             "column": 6,
             "position": 4389
           }
         },
         "isWarning": false
       }
     ]
   }
 ]
}
```

**Comment by KonstantinSviridov** [comment]
fixed in https://github.com/raml-org/raml-js-parser-2/tree/hotfix-for-1.1.2

**Comment by kevinrenskers** [comment]
Great! Is there a timeline for a new release?

**Comment by KonstantinSviridov** [comment]
@ddenisenko , what can you say?

**Comment by ddenisenko** [comment]
Monday or Tuesday next week.

**Comment by kevinrenskers** [comment]
Seems fixed in 1.1.3. Thanks!
